src/main/webapp/assets/modules/contest-detail/contestDetailScoreDirective.js

```diff
@@ -0,0 +1,17 @@
+techlooper.directive('contestDetailScore', function () {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/contest-detail/contestDetailScore.html",
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.